src/commands/moderation/ban.ts

```diff
@@ -0,0 +1,56 @@
+import { botCache, Member, ban, sendMessage } from "./../../../deps.ts";
+import { Embed } from "./../../utils/Embed.ts";
+import { sendEmbed } from "./../../utils/helpers.ts";
+
+botCache.commands.set(`ban`, {
+  name: `ban`,
+  guildOnly: true,
+  arguments: [
+    {
+      name: "member",
+      type: "member",
+      missing: (message) => {
+        return sendMessage(message.channelID, "User not found!");
+      },
+    },
+    {
+      name: "days",
+      type: "number",
+      defaultValue: 0,
+    },
+    {
+      name: "reason",
+      type: "...string",
+      defaultValue: "No reason given",
+    },
+  ],
+  userServerPermissions: ["BAN_MEMBERS"],
+  botServerPermissions: ["BAN_MEMBERS"],
```

Comment:
We should add in `botChannelPermissions: ["VIEW_CHANNEL", "SEND_MESSAGES", "EMBED_LINKS"]`